seed/static/seed/js/controllers/inventory_map_controller.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * :copyright (c) 2014 - 2018, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.
+ * :author
+ */
+angular.module('BE.seed.controller.inventory_map', [])
+  .controller('inventory_map_controller', [
+    '$scope',
+    '$stateParams',
+    'inventory',
+    'urls',
+    'spinner_utility',
+    function ($scope,
+              $stateParams,
+              inventory,
+              urls,
+              spinner_utility) {
+      spinner_utility.show();
+
+      $scope.inventory_type = $stateParams.inventory_type;
+      $scope.data = inventory.results;
+      $scope.pagination = inventory.pagination;
+
+      $scope.geocoded_data = $scope.data.filter(building => building.long_lat);
+      $scope.ungeocoded_data = $scope.data.filter(building => !building.long_lat);
+
+      // Render Map
+      var renderMap = function () {
+        var raster = new ol.layer.Tile({
+          source: new ol.source.OSM()
+        });
+
+        var vector_style = new ol.style.Style({
+          image: new ol.style.Icon({
+            src: urls.static_url + "seed/images/favicon.ico",
```

Comment:
This is slated to change to be a pin vs the NREL SEED icon which was just being used as a placeholder.